moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/planning_component.h

```diff
@@ -236,12 +232,3 @@ class PlanningComponent
   // std::unique_ptr<moveit_msgs::msg::TrajectoryConstraints> trajectory_constraints_;
 };
 }  // namespace moveit_cpp
-
-namespace moveit
```

Comment:
This is deprecated for 22 months. I'd remove it with this PR but I can also revert this change if you don't feel so